transformations/yoda_transform/transformation.py

```diff
@@ -0,0 +1,130 @@
+import itertools
+
+import spacy
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+"""
+Base Class for implementing the different input transformations a generation should be robust against.
+"""
+"""
+Yoda Speak implementation borrowed from here: https://github.com/yevbar/Yoda-Script, with minor mofifications to fix
```

Comment:
Also, could you please reach out to the authors of this repository since the major code has been derived from that work?